src/components/cc-icon/cc-icon.stories.js

```diff
@@ -133,6 +136,9 @@ export const cleverIcons = makeStory(conf, {
     accessibility: {
       enable: false,
     },
+    visual: {
+      enable: false,
```

Comment:
we disable visual testing for Remix & Clever icons because it's irrelevant and very slow (sometimes it even timeouts) since there are a lot of icons.